terraform/modules/simple_server/aws_ami.tf

```diff
@@ -1,17 +1,7 @@
 data "aws_ami" "ubuntu" {
-  filter {
-    name   = "name"
-    values = ["ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-*"]
-  }
-
-  filter {
-    name   = "virtualization-type"
-    values = ["hvm"]
-  }
-
```

Comment:
These are used as search filters to query through all AWS AMIs. Searching by the AMI ID alone is sufficient, these filters were redundant.